Dockerfile.jvm

```diff
@@ -8,7 +8,6 @@ COPY settings.gradle.kts .
 
 COPY eclipse-style.xml eclipse-style.xml
 COPY src src
-COPY api-spec api-spec
```

Comment:
please restore this copy as it will be used in other pr by the api specs generator